internal/core/command.go

```diff
@@ -70,6 +70,16 @@ type WaitFunc func(ctx context.Context, argsI, respI interface{}) error
 
 const indexCommandSeparator = "."
 
+// OverrideRun overrides the CommandRunner function via a builder function.
+func (c *Command) OverrideRun(runBuilder func(command *Command) CommandRunner) {
```

Comment:
May be we should have a singel Override method 
```
cmds.MustFind("instance", "placement-group", "get").Override(placementGroupGetOverride)
```

Reply:
Why do we even need `OverrideRun()` or `Override()` ? 

Can't we just do `myCommand.run = overrideMyCommandRun(myCommand)` ?

Or `myCommand = overrideMyCommand(myCommand)`

A general `OverrideRun()` could be useful if we're doing more than justre reassigning `c.Run`, like logging the fact that something has been overriden. But as is, I don't understand why we need a whole function for `c.Run = runBuilder(c)`.